diff --git a/node_modules/binary-parser-encoder/dist/binary_parser.js b/node_modules/binary-parser-encoder/dist/binary_parser.js
index ee49a70..ef0ac58 100644
--- a/node_modules/binary-parser-encoder/dist/binary_parser.js
+++ b/node_modules/binary-parser-encoder/dist/binary_parser.js
@@ -551,7 +551,7 @@ var Parser = /** @class */ (function () {
     Parser.prototype.compile = function () {
         var importPath = 'imports';
         var ctx = this.getContext(importPath);
-        this.compiled = new Function(importPath, 'TextDecoder', "return function (buffer, constructorFn) { " + ctx.code + " };")(ctx.imports, typeof TextDecoder === 'undefined'
+        this.compiled = new Function(importPath, 'TextDecoder', "return function (buffer, constructorFn) { ' + ctx.code + ' };")(ctx.imports, typeof TextDecoder === 'undefined'
             ? require('util').TextDecoder
             : TextDecoder);
     };
